api/taxonomies/views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import generics
+
+from api.taxonomies.serializers import TaxonomySerializer
+from api.base.views import JSONAPIBaseView
+
+import json
+
+class PlosTaxonomyMixin(JSONAPIBaseView, generics.RetrieveAPIView):
+    permission_classes = (
```

Comment:
Aaaaactually, I'd move the permissions classes into the individual views, as they're ususally set on the view level rather than the mixin level